powerauth-java-crypto/src/test/java/com/wultra/security/powerauth/crypto/lib/util/GenerateVectorDataTest.java

```diff
@@ -61,6 +64,7 @@ public class GenerateVectorDataTest {
     private static ObjectMapper objectMapper;
 
     private static final KeyConvertor KEY_CONVERTOR = new KeyConvertor();
+    private static final Random RANDOM = new Random();
```

Comment:
Can we use `SecureRandom`? `Random` in Java should not be used for any cryptography, even this is just a test.